metricbeat/module/mysql/mysql.go

```diff
@@ -36,7 +56,7 @@ func GetMySQLEnvDSN() string {
 	dsn := os.Getenv("MYSQL_DSN")
 
 	if len(dsn) == 0 {
-		dsn = "root@tcp(127.0.0.1:3306)/"
+		dsn = CreateDSN("tcp(127.0.0.1:3306)/", "root", "")
```

Comment:
Referring to the Getenv call above: I think in other places we take connection information from the env only for tests, and not in the production code. Since we have the option to put env variables in the configuration file, I think we shouldn't also have this in the code.